tests/functional/Bigquery/FullImportTest.php

```diff
@@ -0,0 +1,543 @@
+                'id',
+                'name',
+                'price',
+                'isDeleted',
+            ],
+            false,
+            false,
+            []
+        );
+
+        $importer = new ToStageImporter($this->bqClient);
+        $destinationRef = new BigqueryTableReflection(
+            $this->bqClient,
+            $this->getDestinationDbName(),
+            self::TABLE_TRANSLATIONS
+        );
+        /** @var BigqueryTableDefinition $destination */
+        $destination = $destinationRef->getTableDefinition();
+        $stagingTable = StageTableDefinitionFactory::createStagingTableDefinition($destination, [
+            'id',
+            'name',
+            'price',
+            'isDeleted',
+        ]);
+        $qb = new BigqueryTableQueryBuilder();
+        $this->bqClient->runQuery($this->bqClient->query(
+            $qb->getCreateTableCommandFromDefinition($stagingTable)
+        ));
+        $importState = $importer->importToStagingTable(
+            $source,
+            $stagingTable,
+            $options
+        );
+        $toFinalTableImporter = new FullImporter($this->bqClient);
+        $result = $toFinalTableImporter->importToTable(
+            $stagingTable,
+            $destination,
+            $options,
+            $importState
+        );
+
+        self::assertEquals(3, $destinationRef->getRowsCount());
+    }
+
+    /**
+     * @return Generator<string, array<mixed>>
+     */
+    public function fullImportData(): Generator
+    {
+        $expectedEscaping = [];
+        $file = new CsvFile(self::DATA_DIR . 'escaping/standard-with-enclosures.csv');
+        foreach ($file as $row) {
+            $expectedEscaping[] = $row;
+        }
+        /** @var string[] $escapingHeader */
+        $escapingHeader = array_shift($expectedEscaping); // remove header
+        $expectedEscaping = array_values($expectedEscaping);
+
+        $expectedAccounts = [];
+        $file = new CsvFile(self::DATA_DIR . 'tw_accounts.csv');
+        foreach ($file as $row) {
+            $expectedAccounts[] = $row;
+        }
+        /** @var string[] $accountsHeader */
+        $accountsHeader = array_shift($expectedAccounts); // remove header
+        $expectedAccounts = array_values($expectedAccounts);
+
+        $file = new CsvFile(self::DATA_DIR . 'tw_accounts.changedColumnsOrder.csv');
+        $accountChangedColumnsOrderHeader = $file->getHeader();
+
+        $file = new CsvFile(self::DATA_DIR . 'lemma.csv');
+        $expectedLemma = [];
+        foreach ($file as $row) {
+            $expectedLemma[] = $row;
+        }
+        /** @var string[] $lemmaHeader */
+        $lemmaHeader = array_shift($expectedLemma);
+        $expectedLemma = array_values($expectedLemma);
+
+        // large sliced manifest
+        $expectedLargeSlicedManifest = [];
+        for ($i = 0; $i <= 1500; $i++) {
+            $expectedLargeSlicedManifest[] = ['a', 'b'];
+        }
+
+//        yield 'large manifest' => [
```

Comment:
Proč tu nemáš manifest testy? Pokud ten ToStage importer umí pracovat se seznamem souborů (umí) tak by to mělo jet out of box. (Nevím jak tento large, jestli nemá BQ limit na počet souborů).

Reply:
Jo neviem preco som si myslel ze to nepodporuje manifest teraz resp. sliced pridal som to tam a testy presli